src/pages/components/icon-button.mdx

```diff
@@ -120,10 +120,22 @@ import IconButtonTypesCode from '!!raw-loader!../../examples/button/IconButtonTy
 
 ## API
 
+The IconButton component follows this structure:
+
+```jsx
+<Tooltip> {/* recommended */}
+  <IconButton />
```

Comment:
Please remove the trailing slash. The recommended `aria-label` should be added here.